cmd/kubeadm/app/util/version.go

```diff
@@ -153,3 +166,26 @@ func fetchFromURL(url string, timeout time.Duration) (string, error) {
 	}
 	return strings.TrimSpace(string(body)), nil
 }
+
+// kubeadmVersion returns the version of the client without metadata.
+func kubeadmVersion() (string, error) {
+	kubeadmVersionInfo := version.Get()
```

Comment:
Result of `version.Get()` should be argument to that function. This way we can have proper tests for `kubeadmVersion()` function, which is missing now.